Array Processing Tool/js/arrayHandler.js

```diff
@@ -0,0 +1,92 @@
+class ArrayProcessingTool{
```

Comment:
all methods can be static since you don't have an internal state and you don't need an actual instance of the class.